packages/main/src/components/Grid/index.tsx

```diff
@@ -9,15 +8,12 @@ import React, {
   ReactNode,
   ReactNodeArray,
   Ref,
-  useCallback,
-  useEffect,
-  useMemo,
-  useState
+  useMemo
 } from 'react';
 import { createUseStyles } from 'react-jss';
 import { CommonProps } from '../../interfaces/CommonProps';
-import { JSSTheme } from '../../interfaces/JSSTheme';
 import { styles } from './Grid.jss';
+import { useViewportRange } from '@ui5/webcomponents-react-base/src/hooks/useViewportRange';
```

Comment:
see comment above. You should import from `@ui5/webcomponents-react-base/lib/useViewportRange`